mcp/aruuznigar.py

```
@@ -0,0 +1,424 @@
+def _post(endpoint: str, payload: dict) -> dict:
+    """Make a synchronous POST request to Aruuz Nigar."""
+    url = f"{ARUUZ_NIGAR_BASE_URL}{endpoint}"
+    try:
+        response = httpx.post(url, json=payload, timeout=10.0)
+        response.raise_for_status()
+        return response.json()
+    except httpx.ConnectError:
+        return {"error": "Cannot reach Aruuz Nigar. Is it running?"}
+    except httpx.HTTPStatusError as e:
+        return {"error": f"API error {e.response.status_code}: {e.response.text}"}
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)